connectors/src/connectors/zendesk/temporal/activities.ts

```diff
@@ -400,16 +400,22 @@ export async function syncZendeskTicketsActivity({
 
   await concurrentExecutor(
     tickets,
-    (ticket) =>
-      syncTicket({
+    async (ticket) => {
+      const comments = await zendeskApiClient.tickets.getComments(ticket.id);
+      const users = await zendeskApiClient.users.list();
```

Comment:
Can we fetch all users once for all instead at for every ticket?

Reply:
Oops, sure